incubator/java-spring-boot2/image/project/Dockerfile

```diff
@@ -71,8 +62,14 @@ RUN /project/java-spring-boot2-build.sh package
 ####
 FROM registry.access.redhat.com/ubi8/ubi
 
+ARG artifactId=appsody-spring
```

Comment:
Not Sure these are relevant, we should probably update them along with the labels if we want to keep them.

Reply:
Leave it alone I reckon.

Reply:
The ARGS are used in setting the Labels just below. If we lose the labels we can remove these. I'm not convinced the labels have value to us but maybe we should change them to be more relevant.